pylint/pyreverse/main.py

```diff
@@ -2,17 +2,18 @@
 # For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
 # Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
 
-"""%prog [options] <packages>.
+"""{prog} [options] <packages>.
 
   create UML diagrams for classes and modules in <packages>
 """
```

Comment:
```suggestion
"""Create UML diagrams for classes and modules."""
```
I think argparse is generating this text.

Reply:
No we actually pass this as `usage`. That's why we need to change the `prog` as it using an old way of formatting.

Reply:
Could we pass this as doc instead ?

Reply:
Not sure you what you mean with `as doc`? See https://docs.python.org/3/library/argparse.html

Reply:
Sorry I meant ``description``, thank you for linking the documentation.

Reply:
The automatically generated doc is vastly superior to what we can do:
```
python prog.py -h
usage: prog.py [-h] [--sum] N [N ...]
```

Reply:
Fine with me! I'll change it around!